### PR TITLE
ivikys: set submodule lib/ivikys default branch to iv-work-pool-support-for-slave-work-items and synced HEAD to latest from the origin

### PR DESCRIPTION
**Pull request by HofiOne**
the last missing step to get the latest ivikys fork state in syslog-ng builds

Signed-off-by: Hofi <hofione@gmail.com>

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -2,6 +2,7 @@
         path = lib/ivykis
         url = https://github.com/bazsi/ivykis.git
         ignore = dirty
+	branch = iv-work-pool-support-for-slave-work-items
```
Comment (MrAnno) [inline .gitmodules:5]: Let's use spaces consistently in this file.

**Comment by MrAnno** [comment]
@bazsi Can we use your "master" branch instead of this branch?
Even if we work from an ivykis fork, the `master` branch sounds better as it implies some kind of integration workflow (review, test, QA, etc.).

File: .gitmodules (hunk 1 above)
Comment (HofiOne) [reply .gitmodules:5]: funny, it's made by 

`
git config -f .gitmodules submodule."lib/ivykis".branch iv-work-pool-support-for-slave-work-items
`

I'll update manually than

Comment (MrAnno) [reply .gitmodules:5]: Hm... All of my git configs use tabs instead of spaces. This file was probably edited manually then :(

**Review by MrAnno — APPROVED** [review]

**Comment by HofiOne** [comment]
> @bazsi Can we use your "master" branch instead of this branch?
> Even if we work from an ivykis fork, the `master` branch sounds better as it implies some kind of integration workflow (review, test, QA, etc.).

@bazsi, @MrAnno Let's make it in a different run if you do not mind. I merged this one now to be able to work on https://github.com/syslog-ng/syslog-ng/issues/4571
Thx!

**Pull request merged**